class-auth.php

```diff
@@ -187,8 +211,6 @@ public function get_token( WP_REST_Request $request ) {
 				return $user_id;
 			}
 			$user = get_user_by( 'id', $user_id );
```

Comment:
[`get_user_by()`](https://developer.wordpress.org/reference/functions/get_user_by/) may return `false` on failure and not a `WP_Error` that we're checking below with `is_wp_error()`.

Reply:
Valid, though that's an existing potential issue. I think it'd only come up if someone issued a valid token that related to a user that didn't exist. I suspect an edge case, e.g. if the user has since been deleted. Either way, have made a change here to address that.